pkg/ocp4/oauth/oauth_test.go

```diff
@@ -54,7 +54,7 @@ func TestGenYAML(t *testing.T) {
 	crd, manifests, err := Translate(masterV3.Config.OAuthConfig)
 	require.NoError(t, err)
 
-	CRD := crd.GenYAML()
+	CRD, _ := crd.GenYAML()
 	expectedYaml, _ := ioutil.ReadFile("testdata/expected-bulk-test-masterconfig-oauth.yaml")
```

Comment:
Same thing, do not eat the error with `_` because one day this will fail and it will be a pain in the butt to track down why the test is failing. Or worse why the code is failing in production but the tests are passing.